docs/simreal/simwithreal.rst

```diff
@@ -0,0 +1,64 @@
+.. _`Switching between the Real Robot and the Simulated Robot in this Simulation`:
+
+*****************************************************************************
+Switching between the Real Robot and the Simulated Robot in this Simulation
```

Comment:
The title is a bit confusing (we are not using the real robot _in_ the simulation).
Also, given that I consider the `RealRobot` class deprecated, I'm not sure if we should even mention in the documentation.